### PR TITLE
Rename "image_0" topic back to `PVFramesBGR`, misc sync in R18 configs

### PR DESCRIPTION
**Pull request by Purg**
Update all medical, Kitware & BBN, configs to be synchronized with each other.
* Renamed main image topic back to `PVFramesBGR`.
* Ensure each Kitware and BBN set configs are self-consistent, only differing in the configuration files they refer to that are task specific.
* Add pip/poetry cache mounting into docker runtime container.

### CODE REVIEW

**Comment by Purg** [comment]
Eh, I'm going to keep this targetting `master`. This branch is based on @hdefazio's `fix/pytest` branch that needs to land first.

**Comment by Purg** [comment]
I felt paranoid and am checking diffs, found a couple issues with command names, fixing.

**Comment by Purg** [comment]
OK, *now* this is good to go.

File: tmux/demos/medical/Kitware-M5.yml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        - run_image_timestamp: ros2 run angel_system_nodes image_timestamp_relay --ros-args
+            -r __ns:=${ROS_NAMESPACE}
+            -p image_topic:=PVFramesBGR
+            -p output_topic:=PVFramesBGR_ts
+
+#  - sensor_input_stream:
+#      layout: even-vertical
+#      panes:
+#        # Read sensor input from a HoloLens2 unit using HL2SS.
+#        - hl2ss_bridge: ros2 run angel_system_nodes hl2ss_ros_bridge --ros-args
```
Comment (hdefazio) [inline tmux/demos/medical/Kitware-M5.yml:70]: why is the hl2ss bridge commented out?

**Review by hdefazio — APPROVED** [review]

**Pull request merged**